### DISCUSSION TITLE
Cannot connect to ganache

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2627] [Q&A] [answered]
Error sending request for url and multiple other errors while connecting to Ganache.
![Screenshot (31)](https://github.com/user-attachments/assets/537a37cf-f3ea-4a57-88d4-f5f75b9ef552)

**Comment by EngrPips** [comment]
Hello @prakharmishra3956, Please use `Anvil` instead. I believe Patrick used `Anvil` in the course. 

> You should run `Anvil` in a terminal and leave the terminal running 

> Open a second terminal and run your `forge create` command there and things should work fine.

**Reply by prakharmishra3956** [reply]
I have done that but still the error is same.

**Reply by EngrPips** [reply, marked as answer]
Please show the error, also the rpc-url for Anvil is `http://127.0.0.1:8545`  and not `http://127.0.0.1:7545`

**Reply by prakharmishra3956** [reply]
thanks, it is solved now.

**Reply by EngrPips** [reply]
You are welcome, and Keep crushing it
